ccdaservice/winservice.js

```diff
@@ -0,0 +1,37 @@
+var isWin = /^win/.test(process.platform);
+var f = __dirname +''//serveccda.njs';
+
+if( isWin ){
+	var Service = require('node-windows').Service;
+	//f =  '\\xampp\\htdocs\\openemr\\services\\ccdaservice\\serveccda.njs';
```

Comment:
Just wanted to make sure they couldn't be run by a user going to them in the web browser.